l2geth/eth/api_backend.go

```diff
@@ -267,7 +266,6 @@ func (b *EthAPIBackend) GetTd(blockHash common.Hash) *big.Int {
 }
 
 func (b *EthAPIBackend) GetEVM(ctx context.Context, msg core.Message, state *state.StateDB, header *types.Header) (*vm.EVM, func() error, error) {
-	state.SetBalance(msg.From(), math.MaxBig256)
```

Comment:
Perhaps this should be behind a `!vm.UsingOVM` flag as to not break tests and have a comment that this was removed upstream